src/access/profiling/parser.py

```diff
@@ -46,6 +50,38 @@ def read(self, stream: str) -> dict:
             dict: profiling data.
         """
 
+    def parse_data_series(self, logs: list[str], varname: str, vars: Iterable) -> xr.Dataset:
+        """Given a list of logs containing profiling data, parse the data and return it as a xarray dataset.
+
+        Args:
+            Logs (list): Logs to parse.
+            varname (str): Name of the variable that changes between logs.
+            vars (Iterable): An iterable returning the value of the variable that changes between logs.
+
+        Returns:
+            Dataset: Series profiling data.
+        """
+        datasets = []
+        for var, log in zip(vars, logs):
+            data = self.read(log)
+            datasets.append(
+                xr.Dataset(
+                    data_vars=dict(
+                        zip(
+                            self.metrics,
+                            [
+                                xr.DataArray([data[metric]], dims=[varname, "region"]).pint.quantify(metric.units)
+                                for metric in self.metrics
+                            ],
+                        )
+                    ),
+                    coords={"region": data["region"], varname: [var]},
```

Comment:
Please make the ordering consistent between this line and line 73 so that either `varname` always follows (or always precedes) `region`.

Reply:
Done.